components/brave_wallet/browser/BUILD.gn

```diff
@@ -130,6 +129,21 @@ static_library("ethereum_permission_utils") {
   ]
 }
 
+source_set("constants") {
+  sources = [
+    "brave_wallet_constants.cc",
+    "brave_wallet_constants.h",
+  ]
+
+  deps = [
+    "//base",
+    "//brave/components/brave_wallet/common",
+    "//brave/components/brave_wallet/common:mojom",
+    "//brave/components/resources:strings_grit",
+    "//ui/base:base",
```

Comment:
not sure if this is used anywhere (I don't see it used in `brave_wallet_constants.cc`, maybe its not needed?), but careful as many of the files in `//ui/base` are not available in iOS builds

Reply:
and that dep `//ui/base` is already there when iOS depends on `brave/components/brave_wallet/browser`

Reply:
@darkdh Sounds good, didn't realize it was already part of it :)